### PR TITLE
Feat: Added Ratings component

### PR DESCRIPTION
**Pull request by kjais1720**
Add: Two types of Ratings
- Star ratings
- Badge rating
- 3 sizes for ratings, `ratings-sm`, `ratings-md` & `ratings-lg`

Fix: min width of for code snippets

Preview  link👇
https://deploy-preview-19--tarang-ui.netlify.app/docs/ratings/rating

### CODE REVIEW

File: Docs/Ratings/rating.html [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+                    <span>arang</span> <span>UI</span>
+                </h2>
+            </a>
+        </div>
+        <nav class="tr-nav d-flex">
+            <a href="#" class="tr-btn tr-btn-icon-label tr-btn-link">
+                Docs
+                <i class="fas fa-file"></i>
+            </a>
+            <a href="https://github.com/kjais1720/Tarang-UI" class="tr-btn tr-btn-icon-label tr-btn-link">
```
Comment (shubhamkumar648) [inline Docs/Ratings/rating.html:86]: add _blank attributes , page  open in new tab

**Review by shubhamkumar648 — APPROVED** [review]
other looks good for me !

File: stylesheets/components/ratings.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
+.tr-ratings-wrapper .fa-star{
+    color: var(--tr-warning);
+    /* font-size: var(--fs-sm); */
```
Comment (rahulcs754) [inline stylesheets/components/ratings.css:3]: remove comment code

File: stylesheets/components/ratings.css [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+.fa-star.icon-filled{
+    font-weight: 900;
+}
+
+.fa-star.icon-half-filled::before{
+    font-family: "Font Awesome 5 pro"; 
+	font-weight: 900;
+    content: '\f5c0';
```
Comment (rahulcs754) [inline stylesheets/components/ratings.css:15]: use prettify to align all of your code

**Review by rahulcs754 — COMMENTED** [review]
look good but some small changes

File: Docs/Ratings/rating.html (hunk 1 above)
Comment (kjais1720) [reply Docs/Ratings/rating.html:86]: Thank you for pointing it out, I had added it everywhere but forgot to put it here

File: stylesheets/components/ratings.css (hunk 2 above)
Comment (kjais1720) [reply stylesheets/components/ratings.css]: Thank you for pointing it out, I have fixed it in the latest commit

File: stylesheets/components/ratings.css (hunk 3 above)
Comment (kjais1720) [reply stylesheets/components/ratings.css]: Fixed in the latest commit

**Pull request merged**